client/my-sites/activity/activity-log/index.jsx

```diff
@@ -564,6 +568,13 @@ class ActivityLog extends Component {
 
 const emptyList = [];
 
+const hasBackupProduct = purchases => {
+	return find(
+		purchases,
+		purchase => purchase.active && startsWith( purchase.productSlug, 'jetpack_backup_' )
```

Comment:
Instead of `startsWith( purchase.productSlug, 'jetpack_backup_' )` 
Can we use `isJetpackBackup( purchase )` ?

Reply:
Oh great, that will be much nicer.